03/Category.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 
 function Category(props) {
-    return <h2>Category</h2>
-}
+    const { children } = props;
+    return (
+        <>
+            <h2>Category</h2>
+            <ul>
+                {children}
+            </ul>
+        </>
+    );
+};
```

Comment:
Zobacz, że ten komponent wygląda jak `Cart` - żeby nie dublować to może napiszesz jakiś uniwersalny komponent?

Reply:
zmienione :)